### DISCUSSION TITLE
[IMPLEMENTED] Searching through subfolders

### DISCUSSION THREAD

**Discussion by eovnu87435ds** [OctoFarm/OctoFarm#244] [Implemented]
My printers have probably 500 gcode files saved in octoprint, organized into sub-folders. In octoprint, the search function can look through all subfolders at once. It would be really useful to have a similar search feature in octofarm's file manager!

**Comment by NotExpectedYet** [comment]
I may be misunderstanding here but OctoFarms file search should already be recursive is that not the case?

**Comment by NotExpectedYet** [comment]
I've opened an issue for this it should already be recursive.
